.editorconfig

```diff
@@ -0,0 +1,32 @@
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+trim_trailing_whitespace = true
+guidelines = 120
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.cs]
+indent_size = 4
+
+# Organize usings.
+# dotnet_separate_import_directive_groups = false
+# dotnet_sort_system_directives_first = true
+# file_header_template = unset
+
+dotnet_diagnostic.IDE0003.severity = none # Remove `this` or `Me` qualification
+dotnet_diagnostic.IDE0009.severity = suggestion # Add `this` or `Me` qualification
+
+# dotnet_diagnostic.IDE0005.severity = warning # Remove unnecessary import
```

Comment:
Will turn on the imports rules in a later PR since they would add too much churn to this one.